### ISSUE TITLE
Managers can't be deleted once created

### ISSUE THREAD

**Issue by PostPollux** [armadillica/flamenco#214]
When I create a Flamenco Manager on the cloud under https://cloud.blender.org/flamenco/managers/ it can't be deleted anymore.

I can hit the "Create new Manager (max 3)" button to create new Managers, but there seems to be no way to delete a manager once it had been created. At least I can't find one.

Would be nice to have an option in the web interface to delete Managers. Just to keep things clean...

**Comment by sybrenstuvel** [comment]
Deleting is always hard. What should happen with existing jobs & tasks assigned to that Manager? Should they be deleted too? Archived? What if those jobs aren't in a state that allows archiving (say, they're active), what then? Should we only allow deleting Managers that don't have any jobs/tasks?

**Comment by PostPollux** [comment]
Yes, that`s actually what I was thinking about. As long as there are still projects assigned to the manager it should tell you, that first all projects have to be unlinked before a manager can be deleted. 

I don't really understand what exactly will be deleted if a Manager that has no linked projects would be deleted. If that is some important or crucial stuff it could also warn about that things and ask if you really want to proceed...

I know it's not a super important feature, because everything works without it. But it always feels a bit incomplete if you can create stuff but you can't remove it anymore. In my case I just hit that button to test it and now I always have two managers in the dropdown even if I don't need two.

**Comment by sybrenstuvel** [comment]
You're talking about projects, I was talking about jobs & tasks. Both are important.

**Comment by jhowe-3d** [comment]
I would propose allowing deletion of managers without jobs/tasks or where everything is archived. They would also need to have no linked projects? In my case, I created a second manager when I really shouldn't have, and since then it has created confusion as to what projects were connected to what manager. For me it is worth the time to go through and archive everything and clean it out versus seeing "Junk Manager - Do NOT Use!" every time I bring up the managers page.

**Comment by solosails** [comment]
Hi, I'm totally new to Flamenco and am bit by bit learning how to set stuff up, I am also very surprised to see that 
a) you can only have a maximum of three managers
b) you can't delete them once created

As a new user, it's easy to create a new manager not knowing what you are doing and then find you need to delete it to be able to reset things up better, I now only have one manager left -

**Comment by sybrenstuvel** [comment]
@solosails You can just rename a Manager if you want and reuse it. There is no need to delete it in order to set things up better.

**Comment by jhowe-3d** [comment]
I have to agree with @sybrenstuvel here. I honestly just use one manager and have the others renamed. I know it can be annoying to see the inactive manager in some screens, but the effort required to build a deletion mechanism and remove all of the possible relations is time I would like to see spent elsewhere.

@solosails, the good news is that managers are not like wishes. For most of us, we just use one and never have a need or even a temptation to use the other two! 😊

**Comment by solosails** [comment]
Hi all, ah, thanks for that. Glad to hear they are just nominal controllers and not hardware linked. Thanks!

**Comment by solosails** [comment]
It did occur to me, if the reason for not being able to delete a manager is that they are intrinsically linked to any render jobs or archived jobs, would it not be an easy thing to add a “delete” option once a job is made an archive?
